### ISSUE TITLE
Modify output names to support Terraform 0.12

### ISSUE THREAD

**Issue by julianxhokaxhiu** [traveloka/terraform-aws-waf-owasp-top-10-rules#9]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* ***If you are interested in working on this issue or have submitted a pull request, please leave a comment!***

***

### Summary

```
Initializing modules...
Downloading traveloka/waf-owasp-top-10-rules/aws 0.1.1 for waf...
- waf in .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71
There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.

Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 6, in output "01_sql_injection_rule_id":
   6: output "01_sql_injection_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 11, in output "02_auth_token_rule_id":
  11: output "02_auth_token_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 16, in output "03_xss_rule_id":
  16: output "03_xss_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 21, in output "04_paths_rule_id":
  21: output "04_paths_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 26, in output "06_php_insecure_rule_id":
  26: output "06_php_insecure_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 31, in output "07_size_restriction_rule_id":
  31: output "07_size_restriction_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 36, in output "08_csrf_rule_id":
  36: output "08_csrf_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 41, in output "09_server_side_include_rule_id":
  41: output "09_server_side_include_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.
```

### Module Version

0.1.1

### Terraform Version and Terraform AWS Provider Version

```
Terraform v0.12.5
+ provider.aws v2.21.1
```

### Affected Resource(s)

outputs.tf

### Terraform Configuration Files

Not required.

### Expected Behavior

It should work fine

### Actual Behavior

It doesn't.

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

Not required.

### References

None

**Comment by rafikurnia** [comment]
Hi @julianxhokaxhiu thanks for reporting the problem you are facing. Also thanks for the PR #10 !

I suspect it is because you are using Terraform 0.12
Unfortunately this module was developed to support Terraform 0.11 only for now.

**Comment by julianxhokaxhiu** [comment]
Hi @rafikurnia, thanks to you for this module. It's really great to see some implementation of OWASP available in Terraform.

Is there no chance to get the PR merged? I think everyone is onboard on 0.12 now, as it brings many functionalities. Maybe start a 0.2.0 version instead? So everyone who is using 0.1 can keep using Terraform 0.11?

Let me know. I'm fine with whatever as long as the PR is merged. I do like to bring upstream the work I do on top of what I use.

Thank you in advance!

**Comment by jamieneo** [comment]
I'd like to see this merged as well. Can use it on a project.
